### DISCUSSION TITLE
Supabase ForeignKey with self-references can not be select/fetch

### DISCUSSION THREAD

**Discussion by indrazm** [supabase/supabase#12847] [Questions]
Hey,

I have a table called categories.
that there is a column called parent which is has self-references or it has fkey to it table (categories).

When i tried to fetch including the foreign key.
it always return like this one.

```
{
  "code": "PGRST200",
  "details": "Searched for a foreign key relationship between 'categoriesTb' and 'parent' in the schema 'public', but no matches were found.",
  "hint": null,
  "message": "Could not find a relationship between 'categoriesTb' and 'parent' in the schema cache"
}
```

<img width="1285" alt="image" src="https://user-images.githubusercontent.com/114550938/223302610-f92a6bd8-0c55-470b-ab60-0d1427a065e9.png">

Is there anything to do ?
Many Thanks!

**Comment by steve-chavez** [comment]
Can you share the supabase-js query?
